install/kubernetes/helm/istio/charts/prometheus/templates/gateway.yaml

```diff
@@ -0,0 +1,43 @@
+{{- if .Values.ingressgateway.enabled -}}
```

Comment:
this is the incorrect check. istio-ingressgateway

Reply:
this is from the prometheus section - each section is separate ( except global )